docs/sphinx/user_guide/build.rst

```diff
@@ -0,0 +1,212 @@
+.. ##
+.. ## Copyright (c) 2017-23, Lawrence Livermore National Security, LLC
+.. ## and RAJA Performance Suite project contributors.
+.. ## See the RAJAPerf/LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _build-label:
+
+*********************************************
+Building the RAJA Performance Suite
+*********************************************
+
+This section will help you build the RAJA Performance Suite code so you can
+start running it. 
+
+.. _build_reqs-label:
+
+============
+Requirements
+============
+
+The primary requirement for building the RAJA Performance Suite are:
+
+- C++ compiler with C++14 support
+- `CMake <https://cmake.org/>`_ version 3.23 or greater when building the HIP back-end, and version 3.20 or greater otherwise.
+
+For the most part, available configuration options and how to enable or 
+disable them are similar to `RAJA build options <https://raja.readthedocs.io/en/develop/sphinx/user_guide/config_options.html#configopt-label>`_. 
+
+Later in this section, we discuss options that are specific to the 
+RAJA Performance Suite.
+
+.. _build_getcode-label:
+
+==================
+Getting the Code
+==================
+
+The RAJA Performance Suite code is hosted on the 
+`GitHub RAJA Performance Suite project <https://github.com/LLNL/RAJAPerf>`_. 
+To get the code, clone the repository into a local working space using the 
+command::
+
+   $ git clone --recursive https://github.com/LLNL/RAJAPerf.git
+
+The ``--recursive`` option is required to pull all RAJA Performance Suite 
+Git *submodules* on which it depends into your local copy of the repository.
+
+After running the ``git clone`` command, a copy of the RAJA Performance Suite
+repository will reside in the ``RAJAPerf`` subdirectory where you ran the 
+clone command. You will be on the ``develop`` branch, which is the default 
+RAJA Performance Suite branch. For example::
+
+  $ cd RAJAPerf
+  $ git branch | grep \*
+  * develop
+
+If you do not pass the ``--recursive`` argument to the ``git clone``
+command, you can also type the following command in the ``RAJAPerf`` 
+directory after cloning::
+
+  $ git submodule update --init --recursive
+
+Either way, the result is the same and you should be good to configure the
+code and build it.
+
+.. note:: * If you are in your local copy of the RAJA Performance Suite repo
+            and you switch to a different repo branch (e.g., you run the 
+            command ``git checkout <different branch name>``), you may need to 
+            run the command ``git submodule update`` to set the Git *submodule
+            versions* to what is used by the new branch. To see if this is 
+            required, the ``git status`` command will indicate whether the
+            submodules are at the proper versions. 
+          * If the *set of submodules* in a new branch is different than the
```

Comment:
Is this bullet covered in the previous?

Reply:
It's actually different. Same submodules but different versions (first bullet) vs. different submodules (second bullet).